### PR TITLE
[port] Gives the supermatter a new look

### PR DESCRIPTION
**Pull request by patpol4**
# Document the changes in your pull request
dreamseeker_BSvhDqS0Wk
7LtTfGjkOs
hCTemRSz1U

also fixes a tyop in one comment

port of https://github.com/BeeStation/BeeStation-Hornet/pull/2663

# Wiki Documentation

the supermatter sprite is completely different
# Changelog

:cl:  Azlan/SomeAngryMiner
imageadd: removes soul from the supermatter
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
but the soul

would rather have the art directors take a crack at making their own versions

**Comment by SomeguyManperson** [comment]
not the turd crystal nOOOOOOOOOOOOOOOOO

**Comment by JamieD1** [comment]
Looks fucking shit

**Comment by UselessTheremin** [comment]
it looks like someone just downscaled a picture of sulfur it is so yucky lookin

**Comment by patpol4** [comment]
idk i like it

**Comment by ChesterTheCheesy** [comment]
Let's make supermatter purple.

**Comment by Jarod1200** [comment]
no

**Comment by ToasterBiome** [comment]
no

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -89,6 +89,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	desc = "A strangely translucent and iridescent crystal."
 	icon = 'icons/obj/supermatter.dmi'
 	icon_state = "darkmatter"
+	layer = 4
```
Comment (adamsong) [inline code/modules/power/supermatter/supermatter.dm:92]: Why change the layer?
Also, use the macros

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (nesisknesis) [reply code/modules/power/supermatter/supermatter.dm:92]: layer change is so that people dont walk over it since its 2 tiles tall, so they walk under it

**Comment by nesisknesis** [comment]
Okay,

To people who don't want this, do you realise the supermatter looks like this?
image

Compared to that ACTUAL downscaled image, this actually looks like a crystal.

Instead of just downvoting or even having the audacity and disrespect to say "Looks fucking shit", saying something constructive would be leagues more helpful, because then I can know what issue you have with it.

I'll make edits to it, but it'll be based on my own observations since i'm not a mind reader.

**Comment by Cartlord** [comment]
This is going to sound extremely stupid, but I feel like a more detailed sprite is worse for the supermatter - as far as I've gathered, it's meant to be extradimensional or some shit, and I think a less detailed sprite is better to get across the idea that your character can't even look at the thing properly.

**Comment by ynot01** [comment]
> To people who don't want this, do you realise the supermatter looks like this? image

yeah it looks like shit up close, i agree

maybe some refining is in order, but who knows if anyone is willing

but having this PR go through is worse than having that shitty sprite imo

**Comment by Cartlord** [comment]
Also, I want to point out that SS13 sprites weren't designed to be blown up that fucking big

**Comment by JamieD1** [comment]
Its too tall, its leaning to one side it does not have the same randomness as the original

**Comment by The-Sun-In-Splendour** [comment]
ss13 contributors on their way to add sprites that do not fit in any way with the rest of the art style for no other reason than "individually it looks good"

**Comment by nesisknesis** [comment]
> ss13 contributors on their way to add sprites that do not fit in any way with the rest of the art style for no other reason than "individually it looks good"

may you show or explain what our art style is so that I may adhere to it in future

**Comment by nesisknesis** [comment]
I will open a separate PR with my edits once they're done

**Pull request closed without merging**